docs/settings/general-infra-logs-ui-settings.asciidoc

```diff
@@ -0,0 +1,17 @@
+`xpack.infra.enabled`:: Set to `false` to disable the Logs and Infrastructure UI plugin {kib}. Defaults to `true`.
+
+`xpack.infra.sources.default.logAlias`:: Index pattern for matching indices that contain log data. Defaults to `filebeat-*`.
+
+`xpack.infra.sources.default.metricAlias`:: Index pattern for matching indices that contain Metricbeat data. Defaults to `metricbeat-*`.
+
+`xpack.infra.sources.default.fields.timestamp`:: Timestamp used to for sorting log entries. Defaults to `@timestamp`.
```

Comment:
Looks like you missed my earlier comment: Say either "used to sort" or "for sorting"